src/flashbar/flash.tsx

```diff
@@ -58,6 +58,7 @@ export const Flash = React.forwardRef(
       content,
       dismissible,
       dismissLabel,
+      statusIconLabel,
```

Comment:
I would recommend to name it `statusIconAriaLabel`, to keep consistent with other ariaLabel prop we have such as `externalIconAriaLabel`

Reply:
See [my note here](https://github.com/cloudscape-design/components/pull/202#discussion_r957406356)

Reply:
One more opinion there – we do not provide a label here to annotate the icon. Icon is our internal implementation detail (we may change it without a public API change).

Unlike FormField or Link components, here we annotate not the icon directly, but the `type` property. 

How about calling this label `typeLabel` or `typeAnnotationLabel` to clarify that this property has to be aligned with value of `type`?

Reply:
I am not sure if I fully understand the idea. Can we localize the type without public api? What is the pro to annotate type but not the icon?

Reply:
This is only about property naming. I think, calling it “typeAriaLabel” would be semantically more correct

Reply:
I've updated this to 'statusIconAriaLabel' per @YueyingLu